akka-cluster-tools/src/main/resources/reference.conf

```diff
@@ -34,6 +34,11 @@ akka.cluster.pub-sub {
   # If specified you need to define the settings of the actual dispatcher.
   use-dispatcher = ""
 
+  # Maximum number of Subscribe/Unsubscribe messages buffered by mediator actor in situation where topic actor is in process of termination.
+  mediator-buffer-size = 1000
+
+  # Maximum number of Subscribe/Unsubscribe messages buffered by each topic actor in situation where group actor is in process of termination.
+  per-topic-actor-buffer-size = 200
```

Comment:
I don't think we need a limited buffer size for this. We know that the stopping will be quick enough to buffer everything.